gui/run.cpp

```diff
@@ -0,0 +1,56 @@
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+#include "App.h"
+#include "MenuApp.h"
+#include "debug.h"
+
+int main() {
+	// setup
+	SDL_Window *window;
+	SDL_Renderer *renderer;
+
+	if (SDL_Init(SDL_INIT_VIDEO) == -1){
```

Comment:
(Style) Add space between closing parentheses and opening brace -> `if (SDL_Init(SDL_INIT_VIDEO) == -1) {`